fastchat/serve/monitor/classify/label.py

```diff
@@ -199,6 +310,15 @@ def find_required_tasks(row):
     assert len(input_data) == len(input_data.uid.unique())
     print(f"{len(input_data)}# of input data just loaded")
 
+    if args.vision:
+        old_len = len(input_data)
+        input_data["image_hash"] = input_data.conversation_a.map(
+            lambda convo: convo[0]["content"][1][0]
+        )
+        input_data["image_path"] = input_data.image_hash.map(
```

Comment:
Where do we get the `image_hash` here?

Reply:
ah i think i created that column beforehand, let me change

Reply:
I guess the way i have it now doesn't support multi-image